### PR TITLE
Automated bulk updates to replace dependabot

### PR DESCRIPTION
**Pull request by fairclothjm**
# Overview
The purpose of this PR is pilot the use of a GH action to do bulk dependency updates. This GH action will run on the 1st of each month and open a PR with the changes.

If this works we can port this over to the vault-worklfows-common repo

### CODE REVIEW

File: .github/workflows/bulk-dep-upgrades.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: Upgrade dependencies
+
+on:
+  schedule:
+    # Runs every 5 mins
+    - cron: '*/5 * * * *'
```
Comment (fairclothjm) [inline .github/workflows/bulk-dep-upgrades.yaml:6]: This is just to test that this works. This workflow should not create another PR if one already exists. Once we verify this works we can change this to `0 0 1 * *` which will run  at 12:00AM on the first of every month

**Review by vinay-gopalan — DISMISSED** [review]
Looks good, thanks for adding this!

**Comment by fairclothjm** [comment]
@vinay-gopalan oops dismissed your review and need another one to merge please

**Review by Zlaticanin — APPROVED** [review]
👍

**Pull request merged**